components/gitpod-db/src/typeorm/entity/db-user-ssh-public-key.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2020 Gitpod GmbH. All rights reserved.
```

Comment:
Nit: Not very important, but when creating new files, the year should be the current year. 😋

```suggestion
 * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
```